linetools/utils.py

```diff
@@ -581,4 +581,60 @@ def overlapping_chunks(chunk1, chunk2):
     if np.min(chunk2) > np.max(chunk1):
         return False
     else:  # overlap exists
-        return True
\ No newline at end of file
+        return True
+
+
+def is_local_minima(a):
+    """For a given 1-d array a, it returns True for local minima
+    and False otherwise. Both edges of array are always False by
+    definition.
+
+    Parameters
+    ----------
+    a : list, 1-d np.array
+        An array of values
+
+    Returns
+    -------
+    boolean, whether the value is a local minima
+
+    Notes
+    -----
+    See also is_local_maxima()
+
+    """
+    a = np.array(a)
+    mask = np.zeros_like(a).astype(bool)
+    for i in range(1, len(a) - 1):  # note that the two edges are always False by definition
```

Comment:
Can we avoid a FOR loop?

I bet we can, e.g.

b = a - np.roll(a,1)
c = a - np.roll(a,-1)

Now do some logic on b and c...